plugins/modules/proxysql_query_rules_fast_routing.py

```diff
@@ -0,0 +1,452 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2017, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: proxysql_query_rules_fast_routing
+author: "Akim Faskhutdinov (@akimrx)"
+short_description: Modifies query rules for fast routing policies using the proxysql admin interface.
+description:
+   - The M(community.proxysql.proxysql_query_rules_fast_routing) module modifies query rules for fast
+     routing policies and attributes using the proxysql admin interface.
```

Comment:
@markuman thanks for the note!
@akimrx , is there a way we can help to finish the module?

Reply:
@Andersson007 

Hi!

At the moment, I do not have time for open source at all, I do not know when I will be able to complete PR, because my son was born and I have a lot of tasks at my main job.

It will be cool if one of the participants completes what I started.

It seems that all that remains here is to add the correct mode for the `-C` flag.

Reply:
@akimrx hi, thanks for the feedback! And my congratulations!:)
OK, I'll put the `help_wanted` label and will change the description a bit, thank you! if you once decide to finish it, it would be great!